cli/src/main/java/com/box/l10n/mojito/cli/command/GithubGetInstallationTokenCommand.java

```diff
@@ -0,0 +1,62 @@
+package com.box.l10n.mojito.cli.command;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.box.l10n.mojito.cli.console.ConsoleWriter;
+import com.box.l10n.mojito.github.GithubClients;
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Component
+@Scope("prototype")
+@Parameters(
+    commandNames = {"github-install-token"},
+    commandDescription = "Retrieves a github installation token for the specified installation")
+public class GithubGetInstallationTokenCommand extends Command {
+
+  @Autowired(required = false)
```

Comment:
`required = false` is not necessary, there should always be an instance, potentially with no clients in it.